packages/compass-collection/src/components/workspace/workspace.module.less

```diff
@@ -66,6 +66,7 @@
     display: flex;
     align-items: center;
     justify-content: center;
+    overflow: auto;
```

Comment:
This updates the styles so that the contents of a workspace tab (CRUD, schema, etc) scrolls properly.